assets/sass/_helper.scss

```diff
@@ -0,0 +1,17 @@
+/*
+ * Hide only visually, but have it available for screen readers:
+ * http://snook.ca/archives/html_and_css/hiding-content-for-accessibility
+ */
+
+// Added as a class because this should to be obvious in HTML
+
+.is-visuallyhidden {
```

Comment:
Just realised: this could be a nice opportunity to start a documentation section called *Accessibility*, and document this there… :D

@petronbot, @TrebBrennan what do you reckon?

Re. names, should the partial be `_helpers.scss` or would `_accessibility.scss` be apt?

(Noting we don’t have to name the partial after the documentation section.)